packages/graphql-codegen-typescript-operations/src/visitor.ts

```diff
@@ -138,17 +145,27 @@ export class TypeScriptDocumentsVisitor extends BaseDocumentsVisitor<
         this.config.immutableTypes,
         this.config.namespacedImportName,
         enumsNames,
-        this.config.enumPrefix,
+        false,
```

Comment:
Is this `false` because we do it separately in `PreResolveTypesProcessor` now? Should we keep the old `this.config.enumPrefix` value here when the original `CodegenPreResolveTypesProcessor` is used?